docs/manual/scala/guide/cluster/DomainModelling.md

```diff
@@ -1,19 +1,121 @@
 # Domain Modelling
 
+We recommend reading [[Event Sourcing and CQRS|ES_CQRS]] as a prerequisite to this section.
+
+In this section we will guide you through all the necessary steps to model an [Aggregate](https://martinfowler.com/bliki/DDD_Aggregate.html), as defined in Domain-Driven Design, using Akka Persistence Typed and following the CQRS principles.
+
+Lagom embraces Event Sourcing and CQRS by promoting the usage of Akka Persistence Typed in combination the CQRS principle. Akka Persistence Typed provides an API for building event sourced actors, but not necessarily for CQRS Aggregates. To build CQRS applications we need to apply a few rules to our design.
+
+We will explain the API and concepts while building a simplified shopping cart model. A full-fledge shopping cart sample can be found on our [samples repository](https://github.com/lagom/lagom-samples/tree/1.6.x/shopping-cart/shopping-cart-scala).
+
+You will learn, how to:
+
+* model a CQRS Aggregate using Akka Persistence Typed
+* prepare its event journal to be queryable using event tags
+* register the model on Akka's Cluster Sharding
+* look up instances on the cluster and iteract with them
+* define serializers for your commands, events and model state
+* understand the main configuation settings and their raison-être
 
 ## Encoding the model
-  * typed protocol - Command, Event and State
-  * defining typed replies and error handling
-  * Enforced replies
-  * Changing behavior - FSM
+
+ We will start by defining the rules of our shopping cart.
+
+* a shopping cart starts empty, gets items added to it and can be either cancelled or checkedout
+* an empty shopping cart can't be checkedout
+* a cancelled or checkedout can't have new items added to it
+* items are simplified to a `name` and a `quantity`
+* when you re-add an item, you override the exisitng by change its quantity
+* you delete an item by updating its quantity to 0
+
+### Typed protocol - Command, Event and State
+
+We will start by define our model in terms of Commands, Events and State.
```

Comment:
Overall I think it is better to not use "we will" but instead be more neutral and directive. For example:

>  Start by defining your model in terms of Commands, Events, and State.